### DISCUSSION TITLE
Column Name in Tables and Login Page

### DISCUSSION THREAD

**Discussion by GuilhermeHirsch** [Budibase/budibase#1811] [Help] [answered]
Is it possible to hide the column name of a table, or change it in the budibase editor? The name format bothers me because it doesn't allow spacing (postgres). If none of this is possible, what do you suggest me to get around it?

How editable is the login page? Is it possible to change the text, the background?

**Comment by mjashanks** [comment, marked as answer]
Hi @GuilhermeHirsch 

Sorry for missing this one! 

1. It's not possible to change the column name in a table at the moment. The only workaround would be to write a custom SQL query, and use a column alias. I've logged a few table improvements [here](https://github.com/Budibase/budibase/discussions/1913), including this one.
2. In the latest version, you can change the logo on the login page, via the `Organisation` section. The background colour can be changed, with a choice of 4 colours, via `Theming`.

**Reply by GuilhermeHirsch** [reply]
Thank you very much for the feedback, @mjashanks!
